engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -2046,6 +2207,46 @@ private void verifyLiveMigrationMapForKVM(Map<VolumeInfo, DataStore> volumeDataS
             if (destStoragePoolVO == null) {
                 throw new CloudRuntimeException("Destination storage pool with ID " + dataStore.getId() + " was not located.");
             }
+
+            if (storageTypeConsistency == null) {
+                storageTypeConsistency = destStoragePoolVO.isManaged();
+            } else if (storageTypeConsistency != destStoragePoolVO.isManaged()) {
+                throw new CloudRuntimeException("Destination storage pools must be either all managed or all not managed");
+            }
+
+            if (!destStoragePoolVO.isManaged()) {
+                if (destStoragePoolVO.getScope() != ScopeType.CLUSTER) {
```

Comment:
In the case of local storage migration, the storage pool scope type is of `ScopeType.HOST;` therefore, this is also breaking for local storage migration, throwing the exception `CloudRuntimeException`.

Reply:
Thanks for pointing this out @GabrielBrascher, added NFS check to fix the bug